src/services/wopi/index.js

```diff
@@ -125,6 +126,9 @@ class WopiFilesContentsService {
 				if (!file) {
 					throw new NotFound('The requested file was not found! (3)');
 				}
+				if (file.size > Configuration.get('WOPI_MAX_FILE_SIZE')) {
+					throw new GeneralError('The requested file exceeds the WOPI file size limit.');
```

Comment:
We want to avoid 500er errors for knowing errors, that we can filter our alerts in grafana for unkown errors of type 50x. Maybe you found a 40x that match also.

Reply:
Ok, then I would suggest that we just use 400 - bad request because the server cannot handle the request. Maybe it's not 100% appropriate but should be sufficient IMHO.